package ru.sapronov.auth.exception;

public class PersonNotFoundException extends RuntimeException {}
